source/clear-linux/get-started/bare-metal-install/cgdisk-manual-install.rst

```diff
@@ -0,0 +1,303 @@
+   partition. The :command:`cgdisk` utility has pre-defined partition
+   types that can be displayed by pressing the :kbd:`L` key at this prompt to
+   show the hex codes you can use. These codes are used to set the correct
+   :abbr:`GUID (Globally unique identifier)` for *GPT partition types*. This
+   is shown in figure 6:
+
+   .. figure:: figures/cgdisk-manual-install-6.png
+      :scale: 50 %
+      :alt: cgdisk - hex codes for partition types
+
+      Figure 6: :guilabel:`cgdisk - hex codes for partition types`
+
+   The codes that you are interested in using for your three partitions are:
+
+   * ef00 - EFI System
+   * 8200 - Linux swap
+   * 8300 - Linux filesystem
+
+#. Since we are creating the EFI boot partition, enter ``ef00`` as
+   the hexcode for this partition and press :kbd:`Enter`.
+
+#. The final field to enter is the partition name. enter ``boot`` and press
+   :kbd:`Enter` to finish setting up the EFI boot partition. You will see that
+   the first partition will be displayed as a 512MiB partition type of
+   ``EFI System`` and a partition name of ``boot`` as shown in figure 7:
+
+   .. figure:: figures/cgdisk-manual-install-7.png
+      :scale: 50 %
+      :alt: cgdisk - boot partition defined
+
+      Figure 7: :guilabel:`cgdisk - boot partition defined`
+
+Create Linux swap partition
+***************************
+
+You are now ready to create the Linux swap partition. You will notice in
+figure 7 that there are 2 areas defined as free space. The first area at the
+top of the list, the 1007.0 KiB free space, is due to starting the previously
+defined EFI boot partition at sector 2048. This is discussed
+in `Rod Smith's Partitioning advice about alignment`_.
+
+#. Move your cursor to highlight the larger free space of 334.8 GiB at the
+   bottom of the partition list before you begin to create the Linux swap
+   partition. This is shown in figure 8:
+
+   .. figure:: figures/cgdisk-manual-install-8.png
+      :scale: 50 %
+      :alt: cgdisk - free space selection
+
+      Figure 8: :guilabel:`cgdisk - free space selection`
+
+#. To create the Linux swap partition, with the largest free space
+   highlighted, select the :guilabel:`[ New ]` button or press the :kbd:`N`
+   key and enter the following values for the Linux swap partition:
+
+   .. code-block:: console
+
+      First sector:  press :kbd:`Enter` to select the default value
+      Size in sectors:  4G
+      Hex code or GUID:  8200
+      Enter new partition name:  swap
+
+   Your :command:`cgdisk` partition list should now look like figure 9.
+
+   .. figure:: figures/cgdisk-manual-install-9.png
+      :scale: 50 %
+      :alt: cgdisk - swap partition defined
+
+      Figure 9: :guilabel:`cgdisk - swap partition defined`
+
+Create Linux filesystem partition
+*********************************
+
+The final partition that you will create is the Linux filesystem partition to
+be used as the root mount point for you |CL| installation.

```

Comment:
Avoid passive voice and empty phrase. Lastly, we must create the the Linux filesystem partition to use it as the root ...

Reply:
done